cmd/podman/create_cli.go

```diff
@@ -74,6 +74,70 @@ func addWarning(warnings []string, msg string) []string {
 	return append(warnings, msg)
 }
 
+// Format supported.
+// podman run --mount type=bind,src=/etc/resolv.conf,target=/etc/resolv.conf ...
+// podman run --mount type=tmpfs,target=/dev/shm ..
+func parseMounts(mounts []string) error {
+	errInvalidSyntax := errors.Errorf("incorrect mount format : should be --mount type=<bind|tmpfs>,src=<host-dir>,target=<ctr-dir>,[option]")
```

Comment:
options not option no?

Reply:
and no commas?  I'd remove them unless they're part of the syntax.

Reply:
They are part of the syntax

Reply:
oops, scratch the comma remarks, I see it below.